common/display/kmsfencehandler.cpp

```diff
@@ -82,6 +90,11 @@ void KMSFenceEventHandler::HandleRoutine() {
     close(kms_fence_);
     kms_fence_ = -1;
   }
+
+  for (NativeSurface *surface : surfaces_) {
+     surface->SetInUse(false);
```

Comment:
https://github.com/01org/IA-Hardware-Composer/blob/master/common/display/displayqueue.cpp#L294

we wait here till the last commit request is handled in case app tries to present content before the last has been handled.